Utilities/Set-FileContent/Set-FileContent.psm1

```diff
@@ -35,6 +35,6 @@ function Set-FileContent
 
         (Get-Content $FilePath) | Foreach-Object { $PSItem -replace $Match, $ReplaceWith } | Set-Content $FilePath
 
-        Write-Host ("Successfully replaced all occurrences of `"$Match`" with `"$ReplaceWith`" in the file `"$FilePath`"!")
+        Write-Verbose "Successfully replaced all occurrences of `"$Match`" with `"$ReplaceWith`" in the file `"$FilePath`"!"
```

Comment:
This is also a case where successful execution would have no output whatsoever without `-Verbose`, IMO this should be Write-Output.